addons/web/static/src/views/calendar/calendar_renderer.scss

```diff
@@ -413,7 +413,7 @@
 
                 @include media-breakpoint-down(md) {
                     width: 50%;
-                    min-width: 12rem;
+                    min-width: 9rem;
```

Comment:
Question: why `min-width` is needed?

Reply:
@stefanorigano Actually we could get rid of the line, but then we would have a different render than the master one.

On Master months are rendered by two, while we would have them one by one, should we go for this?

**MASTER**
![Screenshot 2023-05-05 at 12 03 11](https://user-images.githubusercontent.com/128030743/236429839-60439097-5680-4cb5-be5e-cd6520e9dc44.png)

**MILK if we remove min-width**
![Screenshot 2023-05-05 at 12 01 22](https://user-images.githubusercontent.com/128030743/236429483-77df3398-578f-4607-8549-75384dcac71c.png)

Reply:
If the user is supposed to "tap" on these cells, then we should go for the one column layout (impossible to tap on the smaller cells of the 2col layout). I let you investigate and decide, let me know when done 👍

Reply:
For me one column is definitly better in term of accessibility and navigation, as you mentionned, you cours definitly tap on a wrong cell if the view is too small.

will fix this after lunch